proposals/release-approval-process.md

```diff
@@ -10,7 +10,7 @@ OCI projects need a standard process for making releases so the community of mai
 
 ## Security fixes
 
-Security fix releases MUST use security@opencontainers.org instead of dev@opencontainers.org, but should otherwise follow the standard [list-based voting process](#list-based-voting).
+Security fix releases MUST use security@opencontainers.org instead of dev@opencontainers.org, but should otherwise follow the standard [list-based voting process](#list-based-voting). The security@opencontainers.org email includes all members of the TOB; the TOB will guide the security sensitive release with project maintainers.
```

Comment:
I tried to flesh this out a bit in philips/tob#2, but I'm concerned about two things:

* If you pull in the TOB, you have to explain how the TOB and maintainers will interact (i.e. what does “guide” mean in your sentence?).  I'd rather just have {project}+security@ addresses (e.g. runtime-spec+security@opencontainers.org), and the maintainers can pull in the TOB as they see fit (like always).

* Who gets advanced notification of security fixes?  Do folks who want advanced access apply by mailing the security address?  E.g. “please let us know before pushing security fixes so we can patch our system.  We have $N users, and you can trust us not to leak/abuse the information because $REASONS”.  Where is the list of approved early-contacts maintained (probably not in the public project repo)?